platformio_esp32_envs.ini

```diff
@@ -131,6 +131,7 @@ board                     = esp32_4M
 build_flags               = ${esp32_custom_base_LittleFS.build_flags}
                             -DNO_HTTP_UPDATER
 board_build.partitions    = esp32_partition_app1810k_spiffs2172k.csv
+board_name                = "Espressif Generic ESP32 4M Flash ESPEasy 1810k Code/NO-OTA 2M FS"
```

Comment:
Why board name in the .ini file and not in a (separate) board definition?

Reply:
Because there is only 1 environment using that. But it can also be a separate board definition 😃, will fix.